src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java

```diff
@@ -64,35 +72,22 @@
     public FlowEndpointPayload(@JsonProperty("switch-id") SwitchId switchId,
                                @JsonProperty("port-id") Integer portId,
                                @JsonProperty("vlan-id") Integer vlanId,
+                               @JsonProperty("inner-vlan-id") Integer innerVlanId,
                                @JsonProperty("detect-connected-devices")
                                            DetectConnectedDevicesPayload detectConnectedDevices) {
         super(switchId, portId);
         setVlanId(vlanId);
+        setInnerVlanId(innerVlanId);
+
```

Comment:
empty line

Reply:
em... it looks natural there... removed.